src/hazardous/stream/chacha20.rs

```diff
@@ -307,7 +306,6 @@ impl InternalState {
 		})
 	}
```

Comment:
Both `encrypt()`, `decrypt()` and `keystream_block()` are missing the new `#[must_use]` warnings. Whereas `hchacha20()` and `encrypt_in_place` should have theirs removed.

Reply:
For this one, should we remove must_use because the docs are hidden?
```rs
#[must_use]
#[doc(hidden)]
/// HChaCha20 as specified in the [draft-RFC](https://github.com/bikeshedders/xchacha-rfc/blob/master).
pub fn hchacha20(
	secret_key: &SecretKey,
	nonce: &[u8],
) -> Result<[u8; HCHACHA_OUTSIZE], UnknownCryptoError> {
```

Reply:
Yes, precisely.